designer/client/src/components/withFocus.tsx

```diff
@@ -1,78 +0,0 @@
-import { cx } from "@emotion/css";
```

Comment:
Remove withFocus concept and replace it with a global style https://github.com/TouK/nussknacker/blob/b8dea90affe57ad6fc5080164595dfe88a1b452a/designer/client/src/containers/theme/styles.ts#L231